foodnet/membership/models.py

```diff
@@ -63,39 +67,43 @@ def __str__(self):
 
 class Member(models.Model):
```

Comment:
Ah, a membership can only be tied to one local department at a time (and
yes, I admit it is a little complicated to have a membership with several
users).

2015-06-01 11:07 GMT+02:00 pawciobiel <notifications@github.com>:

> In foodnet/membership/models.py
> <https://github.com/kbhff/foodnet/pull/4#discussion_r31409787>:
>
> > @@ -63,39 +67,43 @@ def __str__(self):
> >
> >  class Member(models.Model):
>
> So there are a few things I had to take into consideration:
>
>    - More than one department can have the same member.
>    - One member can have many users including membership owner. (a school
>    or a houshold; AFAIK there is also a requirement for this to be linked to
>    single address)
>    - The department admin can invite a user to a member and to the
>    correct department (at least for now)
>    - In future membership owner will probably be able to invite users.
>
> I agree it is a bit overcomplicated and I would prefer to simplify it, do
> you have any suggestions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kbhff/foodnet/pull/4/files#r31409787>.
>